hashing/factory/hasherFactory.go

```diff
@@ -11,6 +12,8 @@ func NewHasher(name string) (hashing.Hasher, error) {
 	switch name {
 	case "sha256":
 		return sha256.Sha256{}, nil
+	case "legacyKeccak256":
```

Comment:
use the keccak implementation from hashing/keccak, it should be the same.

Reply:
done